android/app/build.gradle

```diff
@@ -103,7 +103,7 @@ android {
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 1
-        versionName "2.0.2"
+        versionName "2.0.3"
```

Comment:
@nehabluedot should it be 2.0.4?

Reply:
Yes updated app version as per bluedot react plugin version